spec/engines/oasis_search_spec.rb

```diff
@@ -15,7 +15,7 @@
     it 'should return a response' do
       normalized_response = image_search.execute_query
       expect(normalized_response.start_record).to eq(1)
-      expect(normalized_response.end_record).to eq(10)
+      expect(normalized_response.end_record).to eq(9)
```

Comment:
The fixture had one entry related to instagram, which has been removed. Noting the change here in case others think it's problematic for the expected value to be different from the [`size` parameter above](https://github.com/GSA/search-gov/blob/1a8f46e548c1b0020dcdab5c1ca7c26ae1852a08/spec/engines/oasis_search_spec.rb#L10).